test/functional/tools/sample_datatypes_conf.xml

```diff
@@ -14,8 +14,16 @@
     <datatype extension="fastqsolexa" type="galaxy.datatypes.sequence:FastqSolexa" display_in_upload="true" />
     <datatype extension="fastqcssanger" type="galaxy.datatypes.sequence:FastqCSSanger" display_in_upload="true" />
     <datatype extension="fastqillumina" type="galaxy.datatypes.sequence:FastqIllumina" display_in_upload="true" />
-    <datatype extension="sam" type="galaxy.datatypes.tabular:Sam" display_in_upload="true" />
+    <datatype extension="sam" type="galaxy.datatypes.tabular:Sam" display_in_upload="true">
```

Comment:
Ah, I completely forget about this file. Is there any reason this is not a symlink from the original one?